api/reporting/sql/researcher_stats_view.sql

```diff
@@ -0,0 +1,4 @@
+CREATE MATERIALIZED VIEW reporting_local.researcher_stats AS
```

Comment:
Where are these sql files used?

Reply:
Sorry, that's confusing. They're for ad-hoc manual queries I was doing in building the system. I was going to put them into the devops repo, but they came in on this branch since I just moved over all the applicable stuff under `reporting`.